sat/cli/jobstat/parser.py

```diff
@@ -0,0 +1,46 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+"""
+The parser for the jobstat subcommand.
+"""
+import sat.parsergroups
+
+
+def add_jobstat_subparser(subparsers):
+    """Add the jobstat parser to the parent parser.
+    Args:
+        subparsers: The argparse.ArgumentParser object returned by the
+            add_subparsers method.
+    Returns:
+        None
+    """
+    format_options = sat.parsergroups.create_format_options()
+    filter_options = sat.parsergroups.create_filter_options()
+
+    jobstat_parser = subparsers.add_parser(
+        'jobstat',
+        help='Application information',
```

Comment:
Minor, but this is not consistent with the form of our other commands' help text. It should be a short imperative sentence like "Report job and application status."

Reply:
Addressed in [ba96883](https://github.com/Cray-HPE/sat/pull/78/commits/ba9688349d8de17facecf7d17f79ba1f201a67e5), thanks!